node/silkworm/db/access_layer.cpp

```diff
@@ -502,22 +578,62 @@ std::optional<ChainConfig> read_chain_config(mdbx::txn& txn) {
     return ChainConfig::from_json(json);
 }
 
+static Bytes head_header_key() {
+    std::string table_name = db::table::kHeadHeader.name;
+    Bytes key{table_name.begin(), table_name.end()};
+    return key;
+}
```

Comment:
Is this actually needed ?
I see you're using below in write_head_header to buil the slice of the key but this implementation implies :

- transform a c_str into a string
- obtain begin and end iterators to the string
- copy string data into bytes
- return bytes 
- transform bytes into a slice

see my comment below

Reply:
Same as previous